docs/webdav.md

```diff
@@ -0,0 +1,109 @@
+
+### Uploading different file types
+
+The sensenet [Content Repository](content-repository.md) is much more than a simple file system. It can distinguish between different file types. An image for example in a regular file system is a simple file with .png or .jpg extension, an image in the sensenet [Content Repository](content-repository.md) however is an content of Image type, which is derived from the File type. Therefore when using the [Content Repository](content-repository.md) via webdav it is desired to make connections between files with different extensions and [Content Repository](content-repository.md) types. This is done with the same web.config setting that is used by the upload function on the ui:
+
+```xml
+<sensenet>
+  <uploadFileExtensions>
+    <add key=".jpg" value="Image"/>
+    <add key=".jpeg" value="Image"/>
+    <add key=".gif" value="Image"/>
+    <add key=".png" value="Image"/>
+    <add key=".bmp" value="Image"/>
+    <add key=".xaml" value="WorkflowDefinition"/>
+  </uploadFileExtensions>
+```
+
+For example when dropping a file with .png extension the content created in the sensenet [Content Repository](content-repository.md) will be of Image type.
+
+### Configuration
+
+The configuration for webdav can be found in the web.config under the `webdavSettings` section:
+
+```xml
+<sensenet>
+  <webdavSettings>
+    <add key="MockExistingFiles" value="desktop.ini,Thumbs.db,wdmaud.drv,foo,MSGRHU32.ini"/>
+    <add key="WebdavDefaultPageTemplate" value="Internet_Page.html"/>
+  </webdavSettings>
+```
+
+The following options can be set:
+
+- **MockExistingFiles**: some versions of webdav clients shipped with Windows will automatically probe on certain files and will fail to continue if those files are not present. Since these files are never present in a sensenet [Content Repository](content-repository.md), the webdav handler mocks these files as if they existed there to ensure flawless operation. These files may vary from system-to-system, so if you experience any problems with opening sensenet [Content Repository](content-repository.md) folders via webdav it is possible that your webdav client is looking for files not present in the [Content Repository](content-repository.md). Use fiddler or a debug version of sensenet ECM with dbgview to detect this situation and extend this list with the file names to be mocked as existing.
+
+- **WebdavDefaultPageTemplate**: if you upload an aspx file the pagetemplate of the created sensenet Page will be the one specified with this option.
+
+## Custom WebDav provider
+
+It is possible to customize the behavior of the WebDav feature by developing a custom provider. Developers can restrict what users see in Windows Explorer and what they can do with files and folders.
+
+- [How to create a custom WebDav provider](http://community.sensenet.com/tutorials/how-to-create-a-custom-webdav-provider)
```

Comment:
IMHO we should somehow indicate if a community link is yet to be created. Maybe "tag" it somehow?